### PR TITLE
Fix service test and directory structure

### PR DESCRIPTION
**Pull request by lakhoune**
This release fixes the service test failure which was caused by incompatibility between jacoco and java17. Since service tests were always excluded in the docker build this issue was unnoticed until now. 
It also fixes the directory structure which was wrong configured in #10 
Futhermore, tests are now added to the CI pipeling

### CODE REVIEW

File: app/.settings/org.eclipse.jdt.core.prefs [hunk 1]
```diff
@@ -0,0 +1,13 @@
+#
```
Comment (Aran30) [inline app/.settings/org.eclipse.jdt.core.prefs:1]: is this file needed?

**Review by Aran30 — APPROVED** [review]

File: app/.settings/org.eclipse.jdt.core.prefs (hunk 1 above)
Comment (lakhoune) [reply app/.settings/org.eclipse.jdt.core.prefs:1]: yes
